pkg/kubectl/cmd/auth/cani.go

```diff
@@ -19,34 +19,45 @@ package auth
 import (
 	"errors"
 	"fmt"
+	"io"
 	"io/ioutil"
 	"os"
+	"sort"
 	"strings"
 
 	"github.com/spf13/cobra"
 
 	authorizationv1 "k8s.io/api/authorization/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
 	"k8s.io/apimachinery/pkg/api/meta"
 	"k8s.io/apimachinery/pkg/runtime/schema"
+	utilerrors "k8s.io/apimachinery/pkg/util/errors"
 	"k8s.io/cli-runtime/pkg/genericclioptions"
 	authorizationv1client "k8s.io/client-go/kubernetes/typed/authorization/v1"
+	rbacv1helpers "k8s.io/kubernetes/pkg/apis/rbac/v1"
 	"k8s.io/kubernetes/pkg/kubectl/cmd/templates"
 	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+	"k8s.io/kubernetes/pkg/printers"
+	util "k8s.io/kubernetes/pkg/printers/internalversion"
+	"k8s.io/kubernetes/pkg/registry/rbac/validation"
```

Comment:
Ditto.

Reply:
@soltysh  To confirm, we shouldn't use any packages except vendor/* or pkg/kubectl itself?
For example, about this patch, within the packages I import into file cani.go, all the packages I need to remove are:
"k8s.io/kubernetes/pkg/printers"
"util "k8s.io/kubernetes/pkg/printers/internalversion"
"k8s.io/kubernetes/pkg/registry/rbac/validation"
rbacv1helpers "k8s.io/kubernetes/pkg/apis/rbac/v1"
Is that correct? Thanks.

Reply:
that is correct.